MacDown/Code/Application/MPMainController.h

```diff
@@ -11,6 +11,6 @@
 
 @interface MPMainController : NSObject <NSApplicationDelegate>
 
-@property (nonatomic, readonly) MPPreferences *prefereces;
+@property (nonatomic, readonly) MPPreferences *preferences;
```

Comment:
Haha typos happen. Was using this property a bit and in the end after typing "prefereces" all the time decided to rename it 😊